cfme/networks/topology.py

```diff
@@ -0,0 +1,222 @@
+import re
+import attr
+
+from navmazing import NavigateToAttribute
+
+from cfme.modeling.base import BaseCollection, BaseEntity
+from cfme.networks.topology_view import TopologyView
+from cfme.utils.appliance.implementations.ui import CFMENavigateStep, navigator, navigate_to
+from wait_for import wait_for
+
+
+@attr.s
+class TopologyLegend(BaseEntity):
```

Comment:
**REQUIRED**
Here and below please document your classes. What is `element`, what type does it have?